docs/specifying-a-call-to-configure.md

```diff
@@ -55,6 +55,9 @@ A.CallTo(fakeShop).Throws(new Exception());
 // Or limit the calls by return type
 A.CallTo(fakeShop).WithReturnType<string>().Returns("sugar tastes good");
 
+// Or limit the calls to methods that return a value
+A.CallTo(fakeShop).WithNonVoidReturnType().Returns("sugar tastes good");
+

```

Comment:
So my "typesafe" comment above should've suggested indicating that `WithNonVoidReturnType` is good to use _when you don't know the return type at compile time_.